main.cpp

```diff
@@ -69,14 +69,16 @@ static volatile bool g_should_exit;
 
 static void help(FILE *fp) {
     fprintf(fp,
-            "%s [OPTIONS...] <uart>\n\n"
-            "  -h --help                    Print this message\n"
-            "  -b --baudrate                Use baudrate for UART\n"
+            "%s [OPTIONS...] [-s <uart>] [-u <udp_port>]\n\n"
+            "  -s <uart>                    UART device that will be routed\n"
+            "  -u <udp_port>                UDP port that will be routed\n"
```

Comment:
Maybe instead of this you cold parse the last parameter, for UART it will always start with '/' for UDP it will start with anything else.